sw/device/lib/testing/test_framework/ottf_console.c

```diff
@@ -106,18 +106,9 @@ void ottf_console_init(void) {
               .data_phase = kDifSpiDeviceEdgeNegative,
               .tx_order = kDifSpiDeviceBitOrderMsbToLsb,
               .rx_order = kDifSpiDeviceBitOrderMsbToLsb,
-              .device_mode = kDifSpiDeviceModeGeneric,
-              .mode_cfg =
-                  {
-                      .generic =
-                          {
-                              .rx_fifo_commit_wait = kSpiDeviceRxCommitWait,
-                              .rx_fifo_len = kDifSpiDeviceBufferLen / 2,
-                              .tx_fifo_len = kDifSpiDeviceBufferLen / 2,
-                          },
-                  },
+              .device_mode = kDifSpiDeviceModeFlashEmulation,
           }));
-      base_spi_device_stdout(&ottf_console_spi_device);
+      CHECK(false, "spi_device not yet supported as OTTF console.");
```

Comment:
Not a review of this PR but more of a question:

Do you see a way for any of the other SPI modes to work as an OTTF console? In particular, would all of them require constant polling or command processing on the receiving end of messages?

Reply:
Unless we used a GPIO as an interrupt (or similar sideband notification), they would require polling. Command processing is also necessary to update any address that the host should use.

The only FIFO-like interface we have now is the eflash read buffer, and it might go away too.

Reply:
That makes sense, thanks